doc/build-unix.md

```diff
@@ -248,3 +248,26 @@ Additional Configure Flags
 A list of additional configure flags can be displayed with:
 
     ./configure --help
+
+
+
+Cross-compilation
+-------------------
+These steps can be performed on, for example, an Ubuntu VM. The depends system
+will also work on other Linux distributions, however the commands for
+installing the toolchain will be different.
+
+First install the toolchain:
```

Comment:
You are installing the ARM specific toolchain before actually telling the reader that you'll be building for ARM.